### DISCUSSION TITLE
Use excessive capacity on npmplus.

### DISCUSSION THREAD

**Discussion by dalso0418** [ZoeyVid/NPMplus#613] [Q&A] [answered]
Hello. I'm using a good project well.
We have seen cases where the container capacity increases excessively when using the npmplus image that we are currently using for a long time.
![image](https://github.com/ZoeyVid/NPMplus/assets/107911055/c031103d-2436-44de-95b0-0dfe0e426441)

More than 30G capacity is in use in 5 days of operation
When you restart the container, the capacity is restored.

Is there a cause or solution for the increase in capacity?

Thank you, and have a nice day!

*Please understand that this information is written through a translator and there is a mistranslation.

**Comment by dalso0418** [comment]
When I looked up the symptoms, modsec_audit.log is increasing excessively in npmplus /var/log

![image](https://github.com/ZoeyVid/NPMplus/assets/107911055/bca943d9-9bd6-43e3-8d9b-26c7e231fcbf)

**Comment by Zoey2936** [comment, marked as answer]
Please open `/opt/npm/etc/modsecurity/modsecurity-default.conf` and disable `SecAuditLog`

**Reply by dalso0418** [reply]
Thank you for your reply.

SecAuditLog /var/log/modsec_audit.log

Should I say that if I get rid of the above line, it will be output in the docker log and the log will not be saved?

**Reply by Zoey2936** [reply]
yes
